src/widget/wbattery.cpp

```diff
@@ -106,9 +111,9 @@ void WBattery::update() {
                                               m_chargingPixmaps.size())];
             }
             if (minutesLeft == Battery::TIME_UNKNOWN) {
-                setBaseTooltip(tr("Time until charged unknown."));
+                appendBaseTooltip(tr("Time until charged unknown."));
```

Comment:
I would just omit this text. 
If one does not have the time estimation feature, he will probably know it anyway.

Reply:
Yes, I think this text is not very helpful..